scripts/restore-sanitised-database.sh

```diff
@@ -6,6 +6,8 @@ LOCAL_DB_PASSWORD="password"
 LOCAL_DB_USER="postgres"
 LOCAL_DB_NAME="data_store"
 DB_IMAGE="postgres:16.2"
+RESTORE_DB=${1}
```

Comment:
This parameter is kind of confusing for a couple of reasons:

- The name `RESTORE_DB` does not match the true meaning of the variable, which is more `RESTORE_TO_LOCAL_DB`
- The parameter is not a named parameter meaning that usage e.g., `scripts/restore-sanitised-database.sh y` is not intuitive at all
- Separate point that goes beyond this PR, but if the user selects no they do not want to restore to their local database, the default is to sanitise in the cloud which seems like an odd thing to default to, as this is the more dangerous action. The user should be made to explicitly specify that they want to sanitise the DB on AWS